src/lesson9/solution/ihor/SecretArchive.kt

```diff
@@ -0,0 +1,24 @@
+package lesson9.solution.ihor
+
+class SecretArchive {
+    var archive = mutableMapOf<String, Person>()
+    fun mapUpdater(number: String, person: Person) {
```

Comment:
If u replace 2 method with one - at least leave a comment, what this method does)